src/client/src/components/DraggableSidebarItem/index.tsx

```diff
@@ -139,12 +141,13 @@ const DraggableSidebarItem = ({
             </div>
           )}
         </div>
-        <CloseSVG
+        {(typeof totalPageNumber !== 'undefined'? totalPageNumber>1 : true) && <CloseSVG
```

Comment:
Generally it is better to do totalPageNumber !== undefined as @sibille mentioned. 
```suggestion
        {(totalPageNumber !== undefined ? totalPageNumber > 1 : true) && <CloseSVG
```

Reply:
updated my change